histomicstk/features/ComputeIntensityFeatures.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import pandas as pd
+import scipy.stats
+from skimage.measure import regionprops
+
+
+def ComputeIntensityFeatures(im_label, im_intensity,
+                             num_hist_bins=10, rprops=None):
+    """
+    Calculates intensity features from an intensity image.
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+
+    im_intensity : array_like
+        Intensity image
+
+    num_hist_bins: int, optional
+        Number of bins used to computed the intensity histogram of an object.
+        Histogram is used to energy and entropy features. Default is 10.
+
+    rprops : output of skimage.measure.regionprops, optional
+        rprops = skimage.measure.regionprops( im_label ). If rprops is not
+        passed then it will be computed inside which will increase the
+        computation time.
+
+    Coords : array_like
+        A N x 2 coordinate list of a region.
+
+    Returns
+    -------
+    fdata: pandas.DataFrame
+        A pandas dataframe containing the intensity features listed below for
+        each object/label.
+
+    Notes
+    -----
+    List of intensity features computed by this function:
+
+    MinIntensity : float
+        Minimum intensity of object pixels.
+
+    MaxIntensity : float
+        Maximum intensity of object pixels.
+
+    MeanIntensity : float
+        Mean intensity of object pixels
+
+    MedianIntensity : float
+        Median intensity of object pixels
+
+    MeanMedianDifferenceIntensity : float
+        Difference between mean and median intensities of object pixels.
+
+    StdIntensity : float
+        Standard deviation of the intensities of object pixels
+
+    IqrIntensity: float
+        Inter-quartile range of the intensities of object pixels
+
+    MadIntensity: float
+        Median absolute deviation of the intensities of object pixels
```

Comment:
Added [Inter-quartile range](https://en.wikipedia.org/wiki/Interquartile_range) and [median absolute deviation](https://en.wikipedia.org/wiki/Median_absolute_deviation) as more robust measure of spread.